packages/main/src/CheckBox.js

```diff
@@ -124,6 +124,18 @@ const metadata = {
 			type: String,
 		},
 
+		/**
+		 * Determines the <code>aria-label</code>, set on the component root tag.
+		 * @type {string}
+		 * @defaultvalue undefined
```

Comment:
As the property is of type string, perhaps the default value in the docs should be empty string?

Reply:
No, it should be empty string when it is empty string, and when it is undefined it should be undefined.